crm_rma_prodlot_invoice/models/stock_transfer_details.py

```diff
@@ -27,38 +27,46 @@ class StockTransferDetails(models.TransientModel):
 
     @api.multi
     def do_detailed_transfer(self):
-        """
-        When incoming type transfer are made and stock move have serial/lot
+        """ When incoming type transfer are made and stock move have serial/lot
         number, the supplier is assigned to the serial/lot number taken from
         picking.
         @return: do_detailed_transfer boolean results
```

Comment:
I know this was already here, but would you mind fixing this half-Sphinx half-Doxygen format `@return:` to Sphinx `:return:` please?